components/service/glean/src/test/java/mozilla/components/service/glean/storages/MemoryDistributionsStorageEngineTest.kt

```diff
@@ -0,0 +1,279 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean.storages
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.test.core.app.ApplicationProvider
+import kotlinx.coroutines.runBlocking
+import mozilla.components.service.glean.collectAndCheckPingSchema
+import mozilla.components.service.glean.private.Lifetime
+import mozilla.components.service.glean.private.MemoryDistributionMetricType
+import mozilla.components.service.glean.error.ErrorRecording
+import mozilla.components.service.glean.histogram.FunctionalHistogram
+import mozilla.components.service.glean.private.MemoryUnit
+import mozilla.components.service.glean.private.PingType
+import mozilla.components.service.glean.testing.GleanTestRule
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Assert.assertEquals
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers
+import org.mockito.Mockito
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
```

Comment:
Can this be done with AndroidJUnit test runner? I know that the rest of Glean still has a lot of Robolectric but the rest of android-components has already switched over.